rpush.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'net-http-persistent'
   s.add_runtime_dependency 'net-http2', '~> 0.14'
   s.add_runtime_dependency 'jwt', '>= 1.5.6'
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activesupport', '>= 5.0'
```

Comment:
We can discuss this change. Without this, people can still install future rpush versions on Rails 4.2, but there's no guarantee it will and I'd rather prevent issues from being opened for unsupported Rails versions. WDYT?

Reply:
👍 Makes sense to me. Worth noting that in changelog + doing a minor version update

Reply:
Thanks for the feedback!
I'll leave this PR open another week or so since some people might be on holidays still. :)